components/brave_rewards/browser/static_values.h

```diff
@@ -22,7 +22,9 @@ const std::vector<std::string> kOnlyAnonWalletCountries = {
 };
 
 #if defined(OS_ANDROID)
-  const std::map<std::string, bool> kBoolOptions = {};
+  const std::map<std::string, bool> kBoolOptions = {
+      {ledger::option::kClaimUGP, false}
+  };
```

Comment:
It's usually better to put the declaration in the header file and the actual definition in a .cc file. Doing it the way it is here can lead to linker errors or subtle bugs.

Reply:
please file an issue and we can refactor, this one just uses existing flows